src/main/scala/com/google/cloud/spark/bigquery/SparkBigQueryOptions.scala

```diff
@@ -23,10 +28,25 @@ import org.apache.spark.sql.types.StructType
 case class SparkBigQueryOptions(
     tableId: TableId,
     parentProject: String,
+    credentials: Option[String] = None,
+    credentialsFile: Option[String] = None,
     filter: Option[String] = None,
     schema: Option[StructType] = None,
     skewLimit: Double = SparkBigQueryOptions.SKEW_LIMIT_DEFAULT,
     parallelism: Option[Int] = None) {
+
+  def createCredentials: Option[Credentials] =
+    (credentials, credentialsFile) match {
+      case (Some(key), None) =>
+        Some(GoogleCredentials.fromStream(new ByteArrayInputStream(Base64.decodeBase64(key))))
+      case (None, Some(file)) =>
+        Some(GoogleCredentials.fromStream(new FileInputStream(file)))
+      case (None, None) =>
+        None
+      case (Some(_), Some(_)) =>
+        throw new IllegalArgumentException("Credentials can be provided through file " +
+          "name or a Base64-encoded string directly, but not both!")
```

Comment:
Consider rewording to:
"Credentials can be provided using a file or directly through a Base64-encoded string, but not both!"
Or even better:
"Only one of credentials or credentialsFile can be specified in the conf."

Reply:
done.